openedx/features/badging/helpers/badges.py

```diff
@@ -1,25 +1,23 @@
+"""
+Helper methods
```

Comment:
Lets improve docstring by saying `All helper methods specifically related to badges`

or If you have anything better in mind, please add.